commit 59082c477b75960ca589c81738cb5c61cadac73f
Author: Yang Bo <yangbo@linuxdeepin.com>
Date:   Sun Jan 3 18:26:34 2016 +0800

    UBUNTU: SAUCE: (no-up) fix build error with CONFIG_BLK_DEV_INITRD=n
    
    The original aynchronous boot patch (UBUNTU: SAUCE: Make populate_rootfs asynchronous)
    did not take into consideration the case when CONFIG_BLK_DEV_INITRD=n,
    e.g., populate_rootfs_domain becomes undefined. Therefore, add it to
    noinitramfs.c where its use is benign.
    
    Signed-off-by: Tim Gardner <tim.gardner@canonical.com>
    Original-patch-by: Nicolas Pitre <nicolas.pitre@linaro.org>
    Cc: Surbhi Palande <surbhi.palande@canonical.com>
    Signed-off-by: Leann Ogasawara <leann.ogasawara@canonical.com>
    Signed-off-by: Yang Bo <yangbo@linuxdeepin.com>

diff --git a/init/noinitramfs.c b/init/noinitramfs.c
index 267739d..e0f648c 100644
--- a/init/noinitramfs.c
+++ b/init/noinitramfs.c
@@ -22,6 +22,8 @@
 #include <linux/kdev_t.h>
 #include <linux/syscalls.h>
 
+LIST_HEAD(populate_rootfs_domain);
+
 /*
  * Create a simple rootfs that is similar to the default initramfs
  */
